tldw_Server_API/app/core/Web_Scraping/enhanced_web_scraping.py

```
@@ -1117,168 +1118,202 @@ def _split(s: str) -> Set[str]:
+                                    try:
+                                        log_counter("webscraping.crawl.urls_skipped", labels={"reason": "filter_chain"})
+                                    except Exception:
+                                        pass
+                                    logger.debug(f"Skip URL (filters reject): {cand}")
+                                    continue
+                                # Enforce path-depth limit relative to site root to align with expectations
                                 try:
-                                    log_counter("webscraping.crawl.urls_skipped", labels={"reason": "dup_seen"})
+                                    from urllib.parse import urlparse as _urlparse
+                                    _ps_cand = len([seg for seg in (_urlparse(cand).path or '/').split('/') if seg])
                                 except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)